sections/featured-collection.liquid

```diff
@@ -28,24 +28,29 @@
+    assign show_mobile_slider = true
+  endif
 -%}
 
 <div class="color-{{ section.settings.color_scheme }} isolate gradient">
-  <div class="collection page-width{% if section.settings.swipe_on_mobile == true and section.settings.collection.all_products_count > 2 and section.settings.products_to_show > 2 %} page-width-desktop{% endif %} section-{{ section.id }}-padding">
+  <div class="collection page-width{% if show_mobile_slider %} page-width-desktop{% endif %} section-{{ section.id }}-padding">
     {% unless section.settings.title == blank %}
-      <div class="title-wrapper-with-link{% if section.settings.title == blank %} title-wrapper-with-link--no-heading{% endif %}{% if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 %} title-wrapper--self-padded-tablet-down{% endif %} title-wrapper--no-top-margin">
+      <div class="title-wrapper-with-link{% if section.settings.title == blank %} title-wrapper-with-link--no-heading{% endif %}{% if show_mobile_slider %} title-wrapper--self-padded-tablet-down{% endif %} title-wrapper--no-top-margin">
         <h2 class="title{% if section.settings.title == blank %} title--no-heading{% endif %}">{{ section.settings.title | escape }}</h2>
   
         {%- if section.settings.show_view_all and section.settings.swipe_on_mobile and more_in_collection -%}
           <a href="{{ section.settings.collection.url }}" class="link underlined-link large-up-hide">{{ 'sections.featured_collection.view_all' | t }}</a>
         {%- endif -%}
       </div>
     {% endunless %}
-  
     <slider-component class="slider-mobile-gutter">
-      <ul id="Slider-{{ section.id }}" class="grid grid--2-col product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %}{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %}{% if products_to_display == 4 or section.settings.collection == blank %} grid--2-col-tablet{% else %} grid--3-col-tablet{% endif %}{% if section.settings.collection == blank %} grid--4-col-desktop{% endif %}{% if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 %} slider slider--tablet grid--peek{% endif %}" role="list">
+      <ul id="Slider-{{ section.id }}" class="grid product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %}{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %}{% if section.settings.collection == blank %} grid--2-col-tablet-down{% else %} grid--{{ section.settings.columns_mobile }}-col-tablet-down{% endif %}{% if section.settings.collection == blank %} grid--4-col-desktop{% endif %}{% if show_mobile_slider %} slider slider--tablet grid--peek{% endif %}" role="list">
```

Comment:
What I mean is that this is only a PR for the mobile settings and only the mobile settings are being tested. This will be a quick follow up to update the desktop PR which is already merged.

Reply:
We kinda need to test desktop here anyway because we've removed grid-col classes/conditionals from some sections. Mobile settings are different from desktop settings but the code is intertwined. But if we can do it as a fast follow I suppose that's fine.

Reply:
Also not all sections that have grid on desktop have grid on mobile. So there are a few more spots to test on desktop that arent here. I will create a fast follow to ensure it's merged right away 👍

Reply:
https://github.com/Shopify/dawn/pull/1414